airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/JobSubmitter.java

```diff
@@ -133,6 +140,19 @@ void submitJob(final Job job) {
 
           if (output.getStatus() == io.airbyte.workers.JobStatus.SUCCEEDED) {
             persistence.succeedAttempt(job.getId(), attemptNumber);
+
+            final List<StandardWorkspace> standardWorkspaces = configRepository.listStandardWorkspaces(false);
```

Comment:
This seems like an expensive scan. Can we just get the workspace for this specific sync job run?

Reply:
That would be ideal and it was my first option. How ever I never managed to access to the workspace Id here.

Reply:
Done

Reply:
`job.getScope()` is the connection id for a sync, which can be used to retrieve the workspace id.

Reply:
Done